src/dashboard/script.ts

```diff
@@ -0,0 +1,64 @@
+/// <reference lib="dom" />
+
+import { FetchPageTest } from "./api-test";
+
+type User = {
+  pos: number;
+  level: number;
+  xp: number;
+  userid: string;
+  avatar: string;
+  username: string;
+  progress: number[];
+};
+
+let users: User[] = [];
+
+for (let i = 0; i < 10; i++) {
+  const page = await FetchPageTest(i);
+
+  if (page) {
+    users.push(...page);
+  }
+}
```

Comment:
This simply cannot be in production code, has to be replaced with infinite scrolling / a page selector.